src/css/styles.css

```diff
@@ -431,6 +431,55 @@ html {
   font-weight: var(--strong-font-weight);
 }
 
+.definition-title__tooltip-text{
+  font-size:0.5em;
+  font-weight: normal;
+}
+
+.definition-title__question-mark{
+  width: calc(var(--definition-title-size) / 2);
+  /* vertically center the question mark*/
+  position: absolute;
+  top:0;
+  bottom: 0;
+  margin: auto;
+  /* end vertical centering */
+  left:0.2em;  /* space out the question mark */
+  /* to change the color. Generate a "filter" attribute with the desired color code*/
+  /* https://codepen.io/sosuke/pen/Pjoqqp */
+  filter: invert(20%) sepia(20%) saturate(7344%) hue-rotate(220deg) brightness(90%) contrast(93%);
```

Comment:
NOICE!

I mean... the icon is svg, we could just change the SVG but THIS WORKS TOO!